util/util.go

```diff
@@ -263,7 +263,12 @@ func Runtime() string {
 		return "crun"
 	}
 
-	return DefaultRuntime
+	conf, err := config.Default()
+	if err != nil {
+		logrus.Debugf("Error loading container config when searching for local runtime.")
```

Comment:
```suggestion
		logrus.Debugf("Error loading container config when searching for local runtime: %v", err)
```
I wonder whether this error should be fatal.  I didn't check the code but I would assume that we don't get an error if the config does not exist ... but if there's some real config/syntax/permission error here, then it would be very hard to figure out.